src/main/resources/db/changelog/changes/schema.xml

```diff
@@ -184,6 +184,7 @@
     </changeSet>
 
     <changeSet id="update_status_type" author="Anlord241">
+        <validCheckSum>8:429e8ab296a5ea20f0e1bb7bf7a5f389</validCheckSum>
```

Comment:
is it fix for updated DB? Will it work with a new one?

Reply:
This tag adds one more valid checksum, so both old and new chageSets will be valid.

Reply:
So, for a new one, it will just execute changeSet. This fix is only for updated DB, yep